src/main/java/nl/rijksoverheid/mev/gmapi/GezagTransformer.java

```diff
@@ -114,10 +114,12 @@ private void transformGezagForPersoon(final Persoon persoon, final Gezagsrelatie
                 persoon.addGezagItem(gezag);
             }
             case "V" -> {
-                AbstractGezagsrelatie gezag = new Voogdij()
+                Voogdij gezag = new Voogdij()
                     .minderjarige(new Minderjarige().burgerservicenummer(gezagsrelatie.getBsnMinderjarige()))
-                    .addDerdenItem(new Meerderjarige().burgerservicenummer(gezagsrelatie.getBsnMeerderjarige()))
                     .type(TYPE_VOOGDIJ);
+                if(gezagsrelatie.getBsnMeerderjarige() != null && !gezagsrelatie.getBsnMeerderjarige().isEmpty()) {
+                    gezag.addDerdenItem(new Meerderjarige().burgerservicenummer(gezagsrelatie.getBsnMeerderjarige()));
```

Comment:
`gezagsrelatie.getBsnMeerderjarige()` wordt hier drie keer gebruikt; can be extracted into a local variable to improve readability

Reply:
push ik naar main